alpaca/trading/client.py

```diff
@@ -317,6 +321,28 @@ def close_position(
 
         return Order(**response)
 
+    def exercise_option_contract(
```

Comment:
`exercise_options_position` perhaps?

Reply:
or maybe not as this is what we call it in the docs

Reply:
Thank you! agreed to rename!